openedx/core/djangoapps/content/search/api.py

```diff
@@ -473,10 +535,16 @@ def add_with_children(block):
                 status_cb(
                     f"{num_contexts_done + 1}/{num_contexts}. Now indexing course {course.display_name} ({course.id})"
                 )
+                if course.id in keys_indexed:
+                    num_contexts_done += 1
```

Comment:
Skipped courses are still included in the total count and in the `num_contexts_done`. I like that. I think we should do the same for skipped (already-indexed) libraries. Currently, they're excluded altogether and won't be include din the `num_contexts_done` count.